<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASQt_SourceClass</class>
 <widget class="QWidget" name="ASQt_SourceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>ASQt_Source</string>
  </property>
  <widget class="QTabWidget" name="m_Player">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_PlayerTab">
    <attribute name="title">
     <string>Player</string>
    </attribute>
    <widget class="QPushButton" name="m_PauseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>240</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>139</x>
       <y>219</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>SourceVolume</string>
     </property>
     <widget class="QSlider" name="m_SourceVol">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="m_PlayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_StopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>240</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>571</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Playlist</string>
     </property>
     <widget class="QListWidget" name="m_Playlist">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>551</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="m_EQTab">
    <attribute name="title">
     <string>EQ</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="m_ReverbTab">
    <attribute name="title">
     <string>Reverb</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="m_CompressorTab">
    <attribute name="title">
     <string>Compressor</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
